scripts/signature.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import json
+import sys
+
+import pyvelox.pyvelox as pv
+from deepdiff import DeepDiff
+
+
+# Utility to export and diff function signatures.
+
+
+# From https://stackoverflow.com/questions/287871/how-do-i-print-colored-text-to-the-terminal
+class bcolors:
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    BOLD = "\033[1m"
+
+
+def export(args):
+    """Exports Velox function signatures."""
+    if args.spark:
+        pv.register_spark_signatures()
+    else:
+        pv.register_presto_signatures()
+
+    signatures = pv.get_function_signatures()
+
+    # Convert signatures to json
+    jsoned_signatures = {}
+    for key in signatures.keys():
+        jsoned_signatures[key] = [str(value) for value in signatures[key]]
+
+    # Persist to file
+    json.dump(jsoned_signatures, args.output_file)
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    first_signatures = json.load(args.first)
+    second_signatures = json.load(args.second)
+    delta = DeepDiff(
+        first_signatures, second_signatures, ignore_order=True, report_repetition=True
+    )
+    if delta:
+        if "dictionary_item_removed" in delta:
+            # Exit gracefully if a function is removed
```

Comment:
nit: the comment is a bit misleading. Failing with sys.exit(1) does not sound like "exit gracefully" :)